wavepool/models.py

```diff
@@ -32,7 +32,17 @@ def teaser(self):
 
     @property
     def source_divesite_name(self):
-        return 'Industry Dive'
+        """ Return the real divesite source name for the newspost's source
+        """
+        source_dive = 'Industry Dive'
+        for dive_source in DIVESITE_SOURCE_NAMES:
```

Comment:
We can avoid the for loop kindly [ref](https://github.com/industrydive/wavepool/pull/10#discussion_r568037864)